src/components/Demo/DemoCreate/DemoCreateStart/index.tsx

```diff
@@ -37,12 +37,34 @@ export const DemoCreateStart: FunctionComponent = () => {
     }
   };
 
+  const features = [
+    {
+      img: "circle-form.png",
+      title: "Fill up the content for CoO",
+    },
+    {
+      img: "circle-load.png",
+      title: "Issue a CoO via TradeTrust",
+    },
+    {
+      img: "circle-verify.png",
+      title: "Verify it to make sure it’s not tempered with",
+    },
+  ];
+
   return (
     <>
-      <h4 className="mt-8">Create CoO</h4>
-      <p className="mt-8">
-        See how a TradeTrust Document can be issued and provide your bank the assurance of document integrity
-      </p>
+      <p className="mt-8">Here&apos;s what you can do with this Demo:</p>
```

Comment:
why `&apos;`

Reply:
&apos; is ' character

Reply:
yeah why needa specially use html entity? why not `'` directly

Reply:
error  `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`  react/no-unescaped-entities

Reply:
ok but i read that &#39 is the correct html escape entity